packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| type                                                                                              | `text`, `password`, `search`, `tel`, `email`, `url`, `numeric`                                                                                                                                                                                         | No       | `text`                                                    | type of text to be rendered. this will also determine the keyboard type to be show on mobile devices                                                                                                                                                                                  |
+| label                                                                                             | `string`                                                                                                                                                                                                                                               | Yes      |                                                           | Determines the label of an input field. Also used as `aria-label`                                                                                                                                                                                                                     |
+| labelPosition                                                                                     | `top`, `left`                                                                                                                                                                                                                                          | No       | `top`                                                     | Used to identify if the label of the input field will be placed on the top of the input field or left. `left` is only available for large screen devices. When position is `left` then the width of the label is fixed to `120px` and the content is aligned left.                    |
+| placeholder                                                                                       | `string`                                                                                                                                                                                                                                               | No       | Default to the label if label provided else `Enter Value` | Placeholder text to be displayed inside the input field                                                                                                                                                                                                                               |
```

Comment:
In that case shouldn't we make placeholder required? Otherwise if people leave this as it's it might unintentionally creep into UIs as "Enter value" 🤔

Reply:
+1 to @divyanshu013's point, if we really want to make placeholder necessary then we should make the prop mandatory & you feel people will ignore type-check then just throw exception if the prop is not present